script/table_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'active_support/core_ext/digest/uuid'
```

Comment:
Can you add a comment please summarizing what this script is for?

Reply:
deleting this, it was a rough draft of navigating the structure i had ironed out for the table info and for the general process, doesn't do anything anymore